pkg/crc/machine/bundle/metadata.go

```diff
@@ -199,7 +199,8 @@ func (bundle *CrcBundleInfo) GetBundleType() crcPreset.Preset {
 }
 
 func (bundle *CrcBundleInfo) IsOpenShift() bool {
```

Comment:
This will have to be renamed to something else, or it's bound to cause confusion in the future (`IsPodman()` checks for the `podman` preset, `IsMicroShift()` checks for the `microshift` preset, `IsOpenShift()` checks for the `openshift` **and** the `okd` preset - not consistent).
This can be done as a follow-up PR though, as I can't think of a very good name (`IsOpenShiftLike()`? `IsOKDBased()`?)

Reply:
Yes I definitely also felt the naming feels a bit odd here, but to me it is a slightly wider topic about what contexts the code  cares about what preset is used vs. what capabilities the CRC setup has. It would perhaps help if key capabilities were identified and you could call `SupportsSomeCapability()` rather than `IsPodman()`, `IsOpenShift()` etc. when implementing conditional logic that depends on the cluster having that capability.

Reply:
We are definitely on the same line, I was suggesting the same in a private conversation yesterday ;) :
> it might help to move to some capability-based description of the bundle `NeedsPullSecret()`, `NeedsKubeadmin()`, `HasKubeconfig()`